raiden/tests/smart_contracts/test_decoder_netting_channel.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf8 -*-
+import os
+
+from secp256k1 import PrivateKey
+from ethereum import tester
+from raiden.utils import sha3, privatekey_to_address
+from raiden.messages import DirectTransfer, Lock, MediatedTransfer, RefundTransfer
+from raiden.encoding.messages import wrap_and_validate
+from raiden.encoding.signing import GLOBAL_CTX, address_from_key
+
+root_dir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
You mean `get_project_root()` I suppose? Check the [last commit](https://github.com/raiden-network/raiden/pull/165/commits/6033e8feb5e7cb571ec0a7057e1190044d465095)